routes/cds-services.js

```diff
@@ -213,7 +214,9 @@ async function call(req, res, next) {
       return;
     }
   }
+  console.timeEnd('CDS Prefetch');
 
+  console.time('Custome API call and data translate')
   // Alternative to FHIR queries can also be included via the ALT_FHIR_QUERIES env parameter
```

Comment:
Several `console.time()` and `console.timeEnd()` calls that were added are missing a trailing semi-colon at the end of the line.

Reply:
Semi-colons added in commit https://github.com/ccsm-cds-tools/ccsm-cds-service/pull/27/commits/f240846b3dcca92282383b85d3f4d5294789e180.